org.upscayl.app.metainfo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.upscayl.app</id>
```

Comment:
`upscayl.org` doesn't seem to exist, nor is it referenced anywhere below. app-id have to reference an existing domain under the control of the developers. `io.github.upscayl.Upscayl` would be an acceptable app-id

Reply:
@hfiguiere I brought that up too in https://github.com/upscayl/upscayl/issues/133#issuecomment-1505377489 but could we assume that since there is no-one registered for this domain (`upscayl.org`), it's safe to use it?

I still hope developers will claim the domain, but would be sad to see this effort fail because of money investment into domain.

Reply:
`io.github.upscayl.Upscayl` is used elsewhere in the manifest and would be appropriate.

Reply:
NayamAmarshe now owns the upscayl.org domain.

Reply:
`app` is a bad app name for the app-id

Reply:
well it is upscayl's main+only app

Reply:
that's doesn't make it good.

Reply:
throw some ideas in then :laughing: what about `org.upscayl`?

Reply:
that's not allowed

`org.upscayl.Upscayl` . Similar to the suggestion to use github above.

Reply:
IMHO it should be `org.upscayl.Upscayl`. `app` is a bad name.